snap/snapcraft.yaml

```diff
@@ -66,8 +66,8 @@ description: |
   **Cryptography**
   ---
 
-  The snap installs the CUDA keyring, verifying its integrity using `curl` and SHA256 checksums.
-  The keyring also sets up the appropriate source for the DCGM deb package.\
+  The snap installs the CUDA keyring deb package using `curl` and verifies its integrity using SHA256 checksums.
```

Comment:
this isn't really accurate: the snap does not install the keyring, in fact as a user installing the DCGM snap I don't get the keyring at all